cmd/juju/cloud/detectcredentials.go

```diff
@@ -182,7 +178,8 @@ func (c *detectCredentialsCommand) allClouds() (map[string]jujucloud.Cloud, erro
 	for k, v := range personalClouds {
 		clouds[k] = v
 	}
-	if !c.Local {
+	if !c.Local && c.ControllerName != "" {
+		ctxt.Infof("\nLooking for cloud information on controller %q...", c.ControllerName)
 		// If there is a cloud definition for the same cloud both
 		// on the controller and on the client and they conflict,
 		// we want definition from the controller to take precedence.
```

Comment:
I wonder if this warrants a warning?

Reply:
I do not think it's a warning... I think it's information, as in 'we've looked locally, now we are looking on the controller..." Kind of like a progress of processing or a commentary :)